### ISSUE TITLE
NTR: Ferrochelatase activity low (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8300]
New term request Ferrochelatase activity low (elthat_121118174726):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000697
- label: Low tissue ferrochelatase activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:8571955
- def: Concentration or activity of Ferrochelatase (FECH; EC 4.99.1.1) below the lower limit of normal. FECH enzyme can be measured in multiple tissues including leukocytes and cultured fibroblasts. FECH is the terminal enzyme of the heme biosynthetic pathway, and catalyzes the insertion of iron into protoporphyrin to form heme.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8300
- synonyms:
